src/state/authentication/sagas.ts

```diff
@@ -1,6 +1,6 @@
 import { takeLatest, takeEvery, put, call } from 'redux-saga/effects';
```

Comment:
to this and all sagas files - is any particular reason u tach this files in this commit ? if it is connect with some internet connection reason will be better have this in other PR pls,

Reply:
TS fixes were required